src/openms/include/OpenMS/FORMAT/PepXMLFile.h

```diff
@@ -122,6 +122,12 @@ namespace OpenMS
       keep_native_name_ = keep;
     }
 
+    /// sets the preferred fixed modifications
+    void setPreferredFixedModifications(const std::vector<ResidueModification>& mods);
```

Comment:
would it be better to have a const ResidueModification* here? e.g., one that is owned by ModificationsDB?